### ISSUE TITLE
Enhance technical highlights for the pure PyTorch components

### ISSUE THREAD

**Issue by Nic-Ma** [Project-MONAI/MONAI#2428]
**Is your feature request related to a problem? Please describe.**
Got the feedback from Kristopher Kersten:
can any pytorch optimizer be used with MONAI? or does it have to be implemented in the monai.optimizers class?
working with Rhino to convert one of their existing models that uses torch.optim's SGD, and they're not exactly sure how to translate to the Novograd optimizer that's in monai.optimizer, hoping they might just be able to use the torch sgd.
Is this documented somewhere in the MONAI docs?  It wasn't obvious to Rhino, and now that I'm digging in a bit trying to help them it also seems not that obvious.  Or maybe this is just assumed since it's all based on Pytorch.

I will enhance our highlights doc to make it clear.

**Comment by wyli** [comment]
I think this should be addressed in the optimizer class (and the other modules)? not sure why it's about updating the highlight doc

**Comment by Nic-Ma** [comment]
Hi @wyli ,

I think that because we don't have a **base class or something** to say the high-level points of loss, network and optimizer.
Where would you suggest to say something about this?

Thanks.

**Comment by wyli** [comment]
could be module-level docstrings or in this readme https://github.com/Project-MONAI/MONAI/blob/dev/monai/README.md

**Comment by Nic-Ma** [comment]
OK, sounds good.

Thanks for the suggestion.

**Comment by Nic-Ma** [comment]
BTW, I will also enhance the doc-string of workflows since users may be not clear about the input args of our trainer.

Thanks.
